smt/surrogate_models/neural_net/tests/test_neural_net.py

```diff
@@ -0,0 +1,63 @@
+'''
+Author: Steven Berguin <<steven.berguin@gtri.gatech.edu>>
+
+This package is distributed under New BSD license.
+'''
+
+import unittest
+import smt.surrogate_models.neural_net as nn
+
+
+class Test(unittest.TestCase):
```

Comment:
As it is a new surrogate model, it would be great if it could be tested like the other ones (see <code>smt/tests</code> directory) at least by adding a test in the <code>test_all.py</code> file.

Reply:
Agreed. However, in hindsight, I don't think unit tests are necessary for this and decided to keep such functions insight the module they pertain to with #pragma no cover.